src/containers/Forms/Lesson/CreateLessons.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { Grid, Dialog, DialogContent, Button, DialogTitle, DialogActions } from '@material-ui/core';
+import DateFnsUtils from '@date-io/date-fns';
+import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
```

Comment:
Нужно для локализации
```suggestion
import { ru } from 'date-fns/locale';
import { MuiPickersUtilsProvider, KeyboardDatePicker } from '@material-ui/pickers';
```